pdns/recursordist/Makefile.am

```diff
@@ -216,3 +222,14 @@ $(MANPAGES):
 	exit 1
 endif
 endif
+
+if !HAVE_LUA_HPP
+# no lua.hpp on solaris :(
+lua.hpp:
```

Comment:
this file is created at build-time, not distribution time. We could have a `lua_hpp.mk` that is symlinked and included into the makefiles?